icicle/include/icicle/sumcheck/sumcheck.h

```diff
@@ -1,13 +1,32 @@
 #pragma once
 
+#include "icicle/errors.h"
+#include "icicle/program/returning_value_program.h"
+#include "icicle/sumcheck/sumcheck_proof.h"
+#include "icicle/sumcheck/sumcheck_config.h"
 #include "icicle/sumcheck/sumcheck_transcript_config.h"
-
+#include "icicle/backend/sumcheck_backend.h"
```

Comment:
This is not supposed to be included here. Only backends are supposed to include this to register